shard.yml

```diff
@@ -1,8 +1,8 @@
 name: granite
 
-version: 0.22.0
+version: 1.0.0
```

Comment:
I would ensure the Amber team actually wants this to be a `1.0.0` release.

Reply:
@elorest ?

Reply:
Good question @Blacksmoke16. I would say no, I don't think that's warranted. A 0.23 is what I'd expect.